packages/angular/projects/clarity-adoption/src/app/app.component.ts

```diff
@@ -3,5 +3,13 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
+  styles: [
+    `
+      :host header img {
```

Comment:
@bdryanovski Do we want to move this inside a `scss` file?

Reply:
No, I think that is ok for now there.

There will be some design changes coming in the next weeks related to content structure and styles - so will handle it when we need it to